edx_exams/apps/api/v1/urls.py

```diff
@@ -53,8 +53,8 @@
         LatestExamAttemptView.as_view(),
         name='exams-attempt-latest',
     ),
-    path(
-        'instructor_view/attempts',
+    re_path(
```

Comment:
*Note this is a breaking change*

Rather than making this backward compatible and than cleaning it up I plan to open a frontend PR to align with this change. This isn't used by anyone but us yet so that saves us some wasted time.

All the other instructor endpoints provide course id as context, this makes writing the permission class much simpler.